app/src/main/java/com/example/nikit/eventsapp/MainActivity.java

```diff
@@ -10,12 +10,13 @@
 import android.support.v7.app.AppCompatActivity;
 import android.view.MenuItem;
 
-import com.example.nikit.eventsapp.utils.SharedPreferencesUtil;
+import butterknife.BindView;
+import butterknife.ButterKnife;
 
 public class MainActivity extends AppCompatActivity {
-
+    @BindView(R.id.navigation)
+    BottomNavigationView navigation;
```

Comment:
Is this needed ?

Reply:
@nikit19 We are referencing this view in the on create method. We are setting up the listener

Reply:
Okay